core/templatetags/render_form.py

```diff
@@ -0,0 +1,31 @@
+from django import template
+from django.utils.translation import gettext_lazy as _
+
+register = template.Library()
+
+##################################################################################
+# Template Tag that creates standard form inputs based on a template
```

Comment:
Same comment as above

Reply:
Ik moet meer opletten in mn copy pastes wanneer ik snel templates wil hebben in m'n code (ofwel de standaard imports en layouts enzo... XD